layouts/compliance/pci-dss.html

```diff
@@ -0,0 +1,239 @@
+                Solutions Architect to get an expert consultation.
+            </p>
+            <div class="mt-10">
+                <a href="#sa-form" rel="noopener noreferrer" class="btn-primary">Speak with a Solutions Architect</a>
+                <a href="/get-started/" rel="noopener noreferrer" class="btn-secondary">Get Started with Pulumi</a>
+            </div>
+        </div>
+    </section>
+    <section class="mt-4 py-4 px-4 md:px-0">
+        <div class="container mx-auto max-w-4xl">
+            <h3>What is PCI DSS Compliance?</h3>
+            <p>
+                {{ .Params.description }}
+            </p>
+        </div>
+    </section>
+    <section class="mt-4 py-4 px-4 md:px-0">
+        <div class="container mx-auto max-w-4xl">
+            <h3>Key Aspects of PCI DSS Compliance</h3>
+            <ol>
+                <li>
+                    <strong>Security Controls:</strong> Organizations must implement
+                    specific technical and operational security measures to safeguard
+                    cardholder data. This includes requirements like installing
+                    firewalls, encrypting cardholder data, and using antivirus software.
+                </li>
+
+                <li>
+                    <strong>Access Control:</strong> Only authorized personnel should
+                    have access to cardholder data. This involves setting up strong
+                    access control measures, such as unique user IDs and restricting
+                    physical access to sensitive data.
+                </li>
+
+                <li>
+                    <strong>Monitoring and Testing:</strong> Regularly monitor and test
+                    networks to ensure that security controls are functioning correctly
+                    and to identify vulnerabilities. This includes maintaining logs of
+                    all access to network resources and cardholder data.
+                </li>
+
+                <li>
+                    <strong>Information Security Policy:</strong> Organizations must
+                    maintain a policy that addresses information security for employees
+                    and contractors. This includes regular security awareness training.
+                </li>
+
+                <li>
+                    <strong>Regular Audits:</strong> Organizations that process, store,
+                    or transmit credit card information must undergo regular audits to
+                    ensure they are in compliance with PCI DSS requirements. This can
+                    involve self-assessment or external assessments, depending on the
+                    size of the organization and the volume of transactions processed.
+                </li>
+            </ol>
+        </div>
+    </section>
+
+    {{ if $cloud_page }}
+    <section class="mt-4 py-4 px-4 md:px-0">
+        <div class="container mx-auto max-w-4xl">
+            <h3>12 PCI DSS Requirements</h3>
```

Comment:
should there be a dash between PCI and DSS?

Reply:
I thought there was, but when I went looking online I see it referenced everywhere without the dash. We may also want to consider renaming the page to just PCI and drop the DSS. From doing keyword searches, it seems there is a lot more volume when searching aws PCI compliance without the DSS rather than with the DSS.